tests/10-create.js

```diff
@@ -0,0 +1,167 @@
+/*!
+ * Copyright 2023 Digital Bazaar, Inc. All Rights Reserved
+ */
+import {shouldBeBs58, shouldBeMulticodecEncoded} from './assertions.js';
+import chai from 'chai';
+import {
+  checkDataIntegrityProofFormat
+} from 'data-integrity-test-suite-assertion';
+import {createInitialVc} from './helpers.js';
+import {documentLoader} from './documentLoader.js';
+import {endpoints} from 'vc-api-test-suite-implementations';
+import {validVc as vc} from './validVc.js';
+
+const tag = 'ecdsa-2019';
+const cryptosuite = 'ecdsa-2019';
+const {match, nonMatch} = endpoints.filterByTag({
+  tags: [tag],
+  property: 'issuers'
+});
+const should = chai.should();
+
+describe('ecdsa-2019 (create)', function() {
+  checkDataIntegrityProofFormat({
+    implemented: match,
+    notImplemented: nonMatch
+  });
+  describe('ecdsa-2019 (issuer)', function() {
+    this.matrix = true;
+    this.report = true;
+    this.implemented = [...match.keys()];
+    this.notImplemented = [...nonMatch.keys()];
+    this.rowLabel = 'Test Name';
+    this.columnLabel = 'Implementation';
+    for(const [name, {endpoints, implementation}] of match) {
+      describe(name, function() {
+        const [issuer] = endpoints;
+        const verifier = implementation.verifiers.find(
+          v => v.tags.has(tag));
+        let issuedVc;
+        let proofs;
+        const verificationMethodDocuments = [];
+        before(async function() {
+          issuedVc = await createInitialVc({issuer, vc});
+          proofs = Array.isArray(issuedVc?.proof) ?
+            issuedVc.proof.filter(proof => proof.cryptosuite === cryptosuite) :
```

Comment:
This is questionable. An issuer could issue an ecdsa-2019 proof and forget to add the cryptosuite, in that case their implementation would be ignored entirely.

Reply:
@aljones15 thanks I've removed it and I've updated the tests to use proofs.some in tests. Though, I think that if we don't apply filtering and instead use proofs.some() in our tests, it would become challenging to determine when an ECDSA proof fails a test. For instance, if an ECDSA proof fails the test `proof.cryptosuite === cryptosuite`, but some other proof passes this test despite not being an ECDSA proof, the test would pass and it could be confusing.

Reply:
ok so my concern here is that we filter out proofs the implementer might have intended to be tested so perhaps what we need is a compound some such as

```
proof.some(p => (p.cryptosuite === cryptosuite && p.foo === expectedValue))
```

So we don't accidentally assert on another `DataIntegrityProof`. I do agree with you that we do run the risk of having a false positive by asserting on all proofs, but I also think we want faulty proofs to fail. LMK if this compromise would work as you have raised points I didn't think out well enough etc.

Reply:
@aljones15 I think  it is best to leave it as it is for now, given the fact that currently the implementations only issue VCs with a single proof on it and none of the implementations have multiple proofs on a VC. IMO, this does not pose a significant concern at the moment, and we can revisit/address it in the future if it becomes problematic.